invenio_opendata/base/templates/helpers/text/glossary.json

```diff
@@ -1,405 +0,0 @@
-[
```

Comment:
WRT the removal of the old glossary file, it is still used by https://github.com/cernopendata/opendata.cern.ch/blob/master/invenio_opendata/base/views.py#L566